addons/account/tests/test_chart_template.py

```diff
@@ -267,3 +268,40 @@ def test_message_to_accountants(self):
             ('partner_ids', 'in', normal_user.partner_id.ids),
             ('body', 'like', f"%{self.tax_template_1.name}%"),  # we look for taxes' name that have been sent in the message's body
         ]), 0)
+
+    def test_update_taxes_coa_installed_via_fp(self):
+        """ When a chart_template is installed through the fiscal position system its taxes should also be updated. """
+        chart_template_xmlid = 'l10n_lu.lu_2011_chart_1'
+        chart_template_lu = self.env.ref(chart_template_xmlid, raise_if_not_found=False)
+        if not chart_template_lu:
+            self.skipTest("Test skipped because the Luxembourg chart of accounts was not found")
```

Comment:
You should create a CoA (and an xmlid going with it) for your test rather than depending on an existing one like this. A bit more verbose, but waaay more robust, as the test will be able to run all the time. Plus, a change in the LU CoA won't risk breaking the test.